### PR TITLE
[exporter/loadbalancing] Fix tests

### PR DESCRIPTION
**Pull request by mx-psi**
**Description:** 

Fix loadbalancing tests. Split off from #17044

**Link to tracking Issue:** Relates to #17037

### CODE REVIEW

**Review by TylerHelmuth — APPROVED** [review]

File: exporter/loadbalancingexporter/log_exporter_test.go [hunk 1]
```diff
@@ -195,7 +195,7 @@ func TestConsumeLogsUnexpectedExporterType(t *testing.T) {
 
 	// verify
 	assert.Error(t, res)
-	assert.EqualError(t, res, fmt.Sprintf("unable to export logs, unexpected exporter type: expected *exporter.Logs but got %T", newNopMockExporter()))
```
Comment (bogdandrutu) [inline exporter/loadbalancingexporter/log_exporter_test.go:198]: why this change? Something is off...

Comment (codeboten) [reply exporter/loadbalancingexporter/log_exporter_test.go:198]: this would have changed here: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16830/files?file-filters%5B%5D=.go&show-deleted-files=true&show-viewed-files=true

**Review by codeboten — APPROVED** [review]

**Comment by bogdandrutu** [comment]
Fixed in a separate PR.
